package.json

```diff
@@ -36,9 +39,14 @@
   "devDependencies": {
     "@types/jest": "^26.0.20",
     "@types/mssql": "^6.0.7",
+    "eslint": "^7.27.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.23.3",
+    "eslint-plugin-prettier": "^3.4.0",
     "fastify": "^3.11.0",
     "jest": "^26.6.3",
-    "mssql": "^6.3.1"
+    "mssql": "^7.1.0",
+    "prettier": "^2.3.0"
   },
   "peerDependencies": {
     "mssql": ">=6.0.0"
```

Comment:
It is compatible with 6.x. The change that broke the tests is that since 7.x the default value for `trustServerCertificate` is false, so the tests were failing because of an unsigned certificate.